x/gov/context.go

```diff
@@ -0,0 +1,56 @@
+package gov
+
+import (
+	"context"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/x"
+)
+
+type contextKey int // local to the gov module
+
+const (
+	contextKeyGov contextKey = iota
+)
+
+// withElectionSuccess is a private method, as only this module
+// can add a gov signer
+func withElectionSuccess(ctx weave.Context, ruleID []byte) weave.Context {
+	val, _ := ctx.Value(contextKeyGov).([]weave.Condition)
+	if val == nil {
```

Comment:
No need for this check. The last line works for both cases. `append` accepts `nil` as the first argument.